src/database/db.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
If I wouldn't have known:
- that you used json-server
- found the port number you selected
- how to start the json server
then I wouldn't know how to make "backend" running.

Do you have some ideas what could be done to make this easier for people not knowing your app to set it up? Also for yourself if you come back in a few weeks ;)

Reply:
I can add a script to npm like "npm start json-server"?

Reply:
Then please do it.
Also, information in README how to run the application is useful. So someone knows that he has to run this command in the first place.